care/apps/patients/filters.py

```diff
@@ -53,15 +53,24 @@ class PatientFilter(filters.FilterSet):
     facility_type = filters.ModelMultipleChoiceFilter(
         field_name="patientfacility__facility__facility_type", queryset=facility_models.FacilityType.objects.all()
     )
-    facility_owned_by = filters.ModelMultipleChoiceFilter(
+    ownership_type = filters.ModelMultipleChoiceFilter(
         field_name="patientfacility__facility__owned_by", queryset=commons_models.OwnershipType.objects.all()
     )
     patient_status = filters.MultipleChoiceFilter(
         field_name="patient_status",
-        # method="filter_patient_status",
         choices=patient_constants.PATIENT_STATUS_CHOICES,
     )
 
+    def filter_queryset(self, queryset):
+        qs = super().filter_queryset(queryset)
+        return qs.annotate(
+            facility_status=F("patientfacility__patient_status__name"),
+            facility_name=F("patientfacility__facility__name"),
+            facility_type=F("patientfacility__facility__facility_type__name"),
```

Comment:
same for the remaining 3s

Reply:
filter is for ids only, but for sending data on get,
 we are sending the names corresponding to the ids

Reply:
than it should be view get_queryset function, instead of here.